joomla/joomla-cms
# Support minimum stability for com_joomlaupdate

### Summary of Changes

Implement support for minimum stability for the core.

### Testing Instructions

- Switch to the testing update server (as of writing this it points to an RC update)
- check for updates
- `no downlaods found` message
- apply patch
- switch mimimum stablilty to RC in the updater config
- check for updates again.

### Expected result

Joomla offeres the RC version to update

### Actual result

![image](https://user-images.githubusercontent.com/2596554/62827035-54606400-bbc7-11e9-9ead-31e6cabda458.png)

### Documentation Changes Required

New option needs to be added to the com_joomlaupdate config docs page.

### Additional changes

- alpha order the language strings in com_joomlaupdate
- extend the list of possible issues for no download URL with minimum stability
- shorten the generation of $cache_timeout into one line.
